### PR TITLE
Package Dependencies

### PR DESCRIPTION
**Pull request by agriffard**
https://github.com/graphql/graphiql/releases/tag/graphiql%401.8.8
react 18
typescript
webpack

### CODE REVIEW

**Review by Skrypt — APPROVED** [review]

**Comment by jptissot** [comment]
Make sure react 18 is non breaking.

**Comment by Skrypt** [comment]
https://github.com/graphql/graphiql/blob/e6daf88589926af2f3be38883c203857a9b014b5/packages/graphiql/package.json#L60-L62

**Comment by jptissot** [comment]
https://github.com/otakustay/react-diff-view/blob/master/package.json#L99

File: src/OrchardCore.Modules/OrchardCore.AuditTrail/package.json [hunk 1]
```diff
@@ -5,29 +5,29 @@
   "license": "MIT",
   "description": "A diff module for OrchardCore.",
   "dependencies": {
-    "bootstrap": "4.6.1",
+    "bootstrap": "5.1.3",
```
Comment (jptissot) [inline src/OrchardCore.Modules/OrchardCore.AuditTrail/package.json:8]: Update to bs5 ?

Comment (Skrypt) [reply src/OrchardCore.Modules/OrchardCore.AuditTrail/package.json:8]: That's because we will merge the bootstrap 5 admin PR this week.

**Pull request merged**